src/daemon/org.projectatomic.rpmostree1.policy

```diff
@@ -30,6 +30,17 @@
     </defaults>
   </action>
 
+  <action id="org.projectatomic.rpmostree1.makecache">
```

Comment:
We already have `org.projectatomic.rpmostree1.repo-refresh`...one thing we need to keep in mind is that it's not necessary for polkit actions to be 1-1 with dbus calls.   The more polkit actions we have, the more anyone locking down a system needs to think about them.

Reply:
Good point. I'll merge it with `repo-refresh`.